lib/rspec/core/option_parser.rb

```diff
@@ -291,6 +291,8 @@ def initialize_project_and_exit
     def bisect_and_exit(argument)
       RSpec::Support.require_rspec_core "bisect/coordinator"
 
+      # This might be the best spot to mutate ENV['SPEC_OPTS']
+      # if we do that.
```

Comment:
It's not obvious to me what's better about this spot then the spot you already have it. Can you explain?